clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -4071,6 +4081,168 @@ void handleFailure(Throwable throwable) {
         return new AlterClientQuotasResult(Collections.unmodifiableMap(futures));
     }
 
+    @Override
+    public DescribeUserScramCredentialsResult describeUserScramCredentials(List<String> users, DescribeUserScramCredentialsOptions options) {
+        final KafkaFutureImpl<Map<String, UserScramCredentialsDescription>> future = new KafkaFutureImpl<>();
+        final long now = time.milliseconds();
+        Call call = new Call("describeUserScramCredentials", calcDeadlineMs(now, options.timeoutMs()),
+                new ControllerNodeProvider()) {
```

Comment:
The KIP says `It will be will be sent to the controller, and will return NOT_CONTROLLER if the receiving broker is not the controller.` It says this for both Describe and Alter.  I agree it doesn't seem necessary for Describe.  Would it require an email to the list for notification if we decide to change it?  Or would a KIP update be sufficient?  Do you think we should change it?  @cmccabe any thoughts here as well?

Reply:
Let's see what @cmccabe thinks. We can do whatever we intend to do for similar describe APIs in the KIP-500 world. If we are changing, then we should update the KIP and send a note to the KIP discussion thread.

Reply:
We're planning on getting rid of ControllerNodeProvider as part of KIP-590, so I think these should all just use `leastLoadedNode`.  Right now they set / fetch stuff in ZK directly anyway.

Yes, it will require an email to the mailing list, but I don't think it's a big change

Reply:
Ok, switching Describe to **not** require that it be done on the controller.

> planning on getting rid of ControllerNodeProvider as part of KIP-590 

Leaving Alter alone for now under the assumption that we will fix it as part of the KIP-590 PR.  Let me know if you wish me to change this now instead.